shared/oval/dovecot_enable_ssl.xml

```diff
@@ -3,10 +3,10 @@
     <metadata>
       <title>Enable SSL in Dovecot</title>
       <affected family="unix">
-        <platform>Red Hat Enterprise Linux 6</platform>
+        <platform>multi_platform_rhel</platform>
```

Comment:
The OVAL requires explicitly ```ssl = yes``` to be set in ```/etc/dovecot/conf.d/10-ssl.conf```. But default setting on RHEL/7 has ```ssl = required``` configured (IOW something which is even more strict).

I have filed https://github.com/OpenSCAP/scap-security-guide/issues/1028 to have the OVAL enhanced in the future.

Otherwise (when explicit ```ssl = yes``` is used) it works fine on RHEL-7 (verified). ACK.